### PR TITLE
Java: README.md

### PR DESCRIPTION
**Pull request by cyip10**
https://github.com/Bit-Quill/glide-for-redis/pull/112

### CODE REVIEW

File: java/README.md [hunk 1]
```diff
@@ -1,18 +1,37 @@
...
+
+We've made GLIDE for Redis an open-source project, and are releasing it in Preview to the community to gather feedback, and actively collaborate on the project roadmap. We welcome questions and contributions from all Redis stakeholders.
+This preview release is recommended for testing purposes only.
+
 # Getting Started - Java Wrapper
 
-## Notice: Java Wrapper - Work in Progress
+## System Requirements
+
+The beta release of GLIDE for Redis was tested on Intel x86_64 using Ubuntu 22.04.1, Amazon Linux 2023 (AL2023), and macOS 12.7.
```
Comment (Elen-Ghulam) [inline java/README.md:24]: please add a section for all the supported operating systems

File: java/README.md [hunk 2]
```diff
@@ -106,6 +151,29 @@ CompletableFuture<String> getResponse = client.get("key");
...
+                .useTLS(useSsl)
+                .build();
+
+RedisClusterClient client = RedisClusterClient.CreateClient(config).get();
+
+CompletableFuture<String> setResponse = client.set("key", "foobar");
+assert setResponse.get() == "OK" : "Failed on client.set("key", "foobar") request";
+
+CompletableFuture<String> getResponse = client.get("key");
+assert getResponse.get() == "foobar" : "Failed on client.get("key") request";
```
Comment (Yury-Fridlyand) [inline java/README.md:174]:
```suggestion
assert setResponse.get() == "OK" : "Failed on client.set(\"key\", \"foobar\") request";

CompletableFuture<String> getResponse = client.get("key");
assert getResponse.get() == "foobar" : "Failed on client.get(\"key\") request";
```

Comment (Yury-Fridlyand) [reply java/README.md:174]: Maybe use `GlideString`? APIs with `String` to be removed soon.

File: java/README.md [hunk 3]
```diff
@@ -29,27 +52,49 @@ Software Dependencies:
...
-$ java -version
- openjdk version "11.0.1" 2018-10-16
- OpenJDK Runtime Environment 18.9 (build 11.0.1+13)
- OpenJDK 64-Bit Server VM 18.9 (build 11.0.1+13, mixed mode)
+```bash
+echo $JAVA_HOME
+java -version
 ```
 
 #### Building and installation steps
```
[the remark this answers is not in the document]
Comment (cyip10) [reply java/README.md:100]: I think we should keep this to make sure they are running it on jdk11

[the remark this answers is not in the document]
Comment (Yury-Fridlyand) [reply java/README.md:100]: Good point. So keep that block (java version check) and jdk installation. All the rest installation could be removed.

File: java/README.md [hunk 4]
```diff
@@ -135,14 +100,34 @@ Other useful gradle developer commands:
...
+
+Refer to https://central.sonatype.com/search?q=glide&namespace=software.amazon.glide for your specific system.
+Once set up, you can run the basic examples.
+
+Gradle:
+- Copy the snippet and paste it in the `build.gradle` dependencies section.
+
+Maven (AARCH_64) specific.
+- **IMPORTANT** must include a `classifier` block. Please use this dependency block instead and add it to the pom.xml file.
+```bash
```
Comment (Yury-Fridlyand) [inline java/README.md:114]:
```suggestion
```xml
```

File: java/README.md [hunk 5]
```diff
@@ -135,14 +100,34 @@ Other useful gradle developer commands:
...
+```
+
 ## Basic Examples
 
-### Standalone Redis:
+### Standalone Valkey:
 
 ```java
-import glide.api.RedisClient;
+import glide.api.ValkeyClient;
```
Comment (Yury-Fridlyand) [inline java/README.md:128]:
```suggestion
import glide.api.GlideClient;
```
Update below too

File: java/README.md [hunk 6]
```diff
@@ -135,14 +100,34 @@ Other useful gradle developer commands:
...
+### Setting up the Driver
+
+Refer to https://central.sonatype.com/search?q=glide&namespace=software.amazon.glide for your specific system.
+Once set up, you can run the basic examples.
+
+Gradle:
+- Copy the snippet and paste it in the `build.gradle` dependencies section.
+
+Maven (AARCH_64) specific.
+- **IMPORTANT** must include a `classifier` block. Please use this dependency block instead and add it to the pom.xml file.
```
Comment (Yury-Fridlyand) [inline java/README.md:113]: Shouldn't we include classifier for mac x64 or linux x64?

File: java/README.md [hunk 7]
```diff
@@ -135,14 +100,34 @@ Other useful gradle developer commands:
...
+
+### Setting up the Driver
+
+Refer to https://central.sonatype.com/search?q=glide&namespace=software.amazon.glide for your specific system.
+Once set up, you can run the basic examples.
+
+Gradle:
+- Copy the snippet and paste it in the `build.gradle` dependencies section.
+
+Maven (AARCH_64) specific.
```
Comment (Yury-Fridlyand) [inline java/README.md:112]: Please also add example in gradle syntax.

**Review by acarbonetto — APPROVED** [review]

**Comment by Yury-Fridlyand** [comment]
Supereded by #395

**Pull request closed without merging**